packages/terra-overlay/src/_contains-polyfill.js

```diff
@@ -0,0 +1,19 @@
+// Source: https://github.com/Financial-Times/polyfill-service/pull/183/files
+// Needed for IE11 and IE10
+if (!document.contains) {
```

Comment:
This might not be an issue, but would we want to check if `contains` already exists on node before adding it? I'm not sure what would happen if another package in the future needed this polyfill, would they just overwrite each other?

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2500/commits/89bfcaa561120db6f21d5c0db11f527933d1a33a